tailwind.config.cjs

```diff
@@ -18,7 +18,9 @@ module.exports = withMT({
       decimal: 'decimal',
       circle: 'circle'
     },
-    extend: {}
+    extend: {
+      'header': "url('src/assets/images/header.png')"
```

Comment:
Why don't you use inline CSS instead? I think using too many custom tailwind class will make it hard to manage when the project grows.